enhancements/machine-api/dedicated-instances.md

```diff
@@ -0,0 +1,184 @@
+---
+title: dedicated-instances
+authors:
+  - "@alexander-demichev"
+reviewers:
+  - "@JoelSpeed"
+  - "@enxebre"
+approvers:
+  - "@JoelSpeed"
+  - "@enxebre"
+creation-date: 2020-09-01
+last-updated: 2020-09-01
+status: provisional
+---
+
+# Dedicated instances
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Make it possible for users to create machines which run as dedicated instances. 
+
+## Motivation
+
+Some organizations need to make sure that their workloads are not hosted on the same physical hardware as others.
+
+### Goals
+
+- Provide automation similar to what Machine API supports for spot instances. 
```

Comment:
"Expose a field in the machine API that enables consumers to choose dedicated tenancy"?